### DISCUSSION TITLE
How do you add stickers?

### DISCUSSION THREAD

**Discussion by jinmoria** [silveredgold/beta-protection#29] [Q&A] [answered]
I've been to the server and clicked 'stickers & configuration' but not sure how to add stickers from here, I can't seem to find the add function. Would appreciate any help.

**Comment by silveredgold** [comment, marked as answer]
Assuming you're using the Beta Censoring backend, that's covered in the [Beta Censoring documentation here](https://silveredgold.github.io/beta-censoring/content/configuration.html#sticker-configuration)
